usdt/Cargo.toml

```diff
@@ -14,14 +14,14 @@ repository = "https://github.com/oxidecomputer/usdt.git"
 # where the public-facing `usdt` crate may pick up different versions of the
 # internal implementation crates.
 [dependencies]
-dtrace-parser = { path = "../dtrace-parser", optional = true, version = "= 0.1.15" }
+dtrace-parser = { path = "../dtrace-parser", optional = true, version = "=0.2.0" }
 serde = "1"
-usdt-impl = { path = "../usdt-impl", default-features = false, version = "= 0.4.1", features = [
+usdt-impl = { path = "../usdt-impl", default-features = false, version = "=0.5.0", features = [
   "des",
 ] }
-usdt-macro = { path = "../usdt-macro", default-features = false, version = "= 0.4.1" }
-usdt-attr-macro = { path = "../usdt-attr-macro", default-features = false, version = "= 0.4.1" }
-dof = { path = "../dof", features = ["des"], version = "= 0.2.0" }
+usdt-macro = { path = "../usdt-macro", default-features = false, version = "=0.5.0" }
+usdt-attr-macro = { path = "../usdt-attr-macro", default-features = false, version = "=0.5.0" }
+dof = { path = "../dof", features = ["des"], version = "=0.3.0" }
 goblin = { version = "0.6", features = ["elf32", "elf64"] }
```

Comment:
noticed that this is pretty down-rev -- do we want to consider updating it at the same time?

Reply:
That's part of #152. The bump changed the API enough that I need to manually update things, so I'll take care of that in a separate PR.